wagtail/admin/templates/wagtailadmin/skeleton.html

```diff
@@ -19,6 +19,7 @@
     {% block branding_favicon %}{% endblock %}
 </head>
 <body id="wagtail" class="{% block bodyclass %}{% endblock %} {% if messages %}has-messages{% endif %} focus-outline-on">
+    <a class="skiplink button" href="#main">Skip to main content</a>
```

Comment:
As mentioned by LB this should be:

```suggestion
    <a class="skiplink button" href="#main">{% trans 'Skip to main content' %}</a>
```

…and I haven’t tried but yep would expect this to look much better on page reloads if it was inside `wrapper`, as LB pointed out in his comment. Wagtail has a lot of synchronous JS loading, so it does make a big difference for this to be done before.